wasmtime/_exportable.py

```diff
@@ -0,0 +1,12 @@
+__all__ = (
+    "Exportable",
+)
+
+import typing
+
+from ._func import Func
+from ._globals import Global
+from ._memory import Memory
+from ._table import Table
+
+Exportable = typing.Union[Func, Table, Memory, Global]
```

Comment:
To match the naming in wasmtime itself, could this be named `Extern`?

Reply:
Ah I see this conflicts with the literal `Extern` type. Could we take a leaf out of Rust's book and do something like `IntoExtern`?

Reply:
I'll look into that. So many things called the same thing... Trying to correlate the WebAssembly spec, undocumented C API, wasmtime API and wasmtime-py names has been... a journey. I see a spreadsheet in my future.